### ISSUE TITLE
A11y_JAWS is reading Incorrect role as image for the tooltips present beside labels "Conversion name" and "Scripting compatability level"

### ISSUE THREAD

**Issue by ShaileshChudawkar** [microsoft/azuredatastudio#24217]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Benchmark; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #DSCT1.0.0; #Element:tooltip; #FTP; #JAWS; #WCAG1.3.1; #ScreenReader; #Win10; #Win32; #NVDA:
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**
### Actual Result: 
JAWS is reading Incorrect role as image for the tooltips present beside labels "Conversion name" and "Scripting compatibility level"
Similar issue is observed with NVDA
### Expected Result:
JAWS should read the correct role as tooltip for the tooltips present beside labels "Conversion name" and "Scripting compatability level"
### Environment Details:
Application Name: Database Schema Conversion Toolkit Azure data studio
Version: 1.43.0 (System user)
Windows Version: Windows 10
OS Build: 19045.2846
ScreenReader: JAWS 2023.2303.144 /NVDA 2023.1
### Repro Steps:
1. Login to vm with appropriate credentials
2. Launch Azure data studio.
3. Tab to 'Manage Connection' and press enter
4. Tab to the dash board for Oracle extension
5. Perform 'Convert Objects'
6. "Provide a name for the conversion" blade should open
7. Tab till the tooltip present beside label "Conversion name"
8. Turn on JAWS
9. Check whether JAWS is reading correct role as tooltip for the tooltip icons present.
### User Impact:
ScreenReader Users will be confused if correct role is not defined for the control.

###  Attachment
![2395677_code snippet image (1)](https://github.com/microsoft/azuredatastudio/assets/143179527/68afbb0f-6388-4f4f-8c35-c0b854bbae56)
[2395677_NVDA.webm](https://github.com/microsoft/azuredatastudio/assets/143179527/6d3d5a8d-be9f-421e-81e8-8329227744d4)
[2395677_A11y_Database Schema Conversion Toolkit_JAWS.webm](https://github.com/microsoft/azuredatastudio/assets/143179527/89da351e-5d6c-4971-972b-1d075437b300)

**Comment by ShaileshChudawkar** [comment]
Primary bug: https://msdata.visualstudio.com/Migration%20Assistant/_workitems/edit/2395677

**Comment by kisantia** [comment]
@ShaileshChudawkar this has now been fixed. Please try the [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch) and verify and close this bug.

**Comment by cpichuka** [comment]
@ShaileshChudawkar , did you retest this issue? we have a s360 item linked this.

**Comment by ShaileshChudawkar** [comment]
Verified the issue at
Version: 1.46.1 (user setup)
Commit: ba29842b81dec01177415e53948ca2168e69c3f8
Date: 2023-10-02T18:14:22.887Z
VS Code: 1.79.2
Electron: 22.3.25
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22621
issue is still repro.

**Comment by kisantia** [comment]
@ShaileshChudawkar please test using the [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch), which is version 1.48.0-insider. Version 1.46.1 does not have the fix.

**Comment by ShaileshChudawkar** [comment]
Verified  the issue at
Version: 1.48.0-insider (user setup)
Commit: 943a1ad98e1bd8e3054df539d88cd3174c6b273a
Date: 2023-11-01T22:48:29.430Z
VS Code: 1.82.0
Electron: 25.8.4
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22621
issue is not fixed. NVDA and JAWS announcing graphic to tooltip. NVDA and JAWS should announce tooltip for icon.
<img width="960" alt="Screenshot 2023-11-02 105542" src="https://github.com/microsoft/azuredatastudio/assets/143179527/81bacfbc-b1db-46e1-a524-b09de189da28">
[JAWS.webm](https://github.com/microsoft/azuredatastudio/assets/143179527/8ef36b77-97cb-48e3-a7c8-2c6e11a248db)
[NVDA..webm](https://github.com/microsoft/azuredatastudio/assets/143179527/be2f7629-81a0-4029-a78a-664af0881a09)

**Comment by kisantia** [comment]
@ShaileshChudawkar The fix is now available, please try again with the [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch).

The info icon is now announced as a tooltip (this is a different extension being shown as an example):
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/70900aa7-3167-48b4-baaf-43f43b3177ba)
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/1d88b0eb-ae7d-4e8d-a33c-0b8bc34cbe75)

**Comment by ShaileshChudawkar** [comment]
#closed; #Regressed:11-10-23;
Verified the issue at
Version: 1.48.0-insider (user setup)
Commit: 6357c042001a66b3e3d8fdb6bc8c8d89018a0e64
Date: 2023-11-08T06:36:15.048Z
VS Code: 1.82.0
Electron: 25.8.4
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22631
issue is fixed. Closing the bug.
![Screenshot (240)](https://github.com/microsoft/azuredatastudio/assets/143179527/d0a5d09e-d13b-4aa6-b9df-f873c43cf916)
[NVDA. (2).webm](https://github.com/microsoft/azuredatastudio/assets/143179527/ee05e82a-efa8-489e-8ed5-d79bb6fca1af)
[JAWS (2).webm](https://github.com/microsoft/azuredatastudio/assets/143179527/dc085631-ed43-4623-9f39-a74c4bdf0e7a)
